examples/Makefile

```diff
@@ -35,4 +35,15 @@ test:
 	cd generic; make test
 	cd multi_thread; make test; sleep 10
 	kubectl describe po test-pod-8
-	cd exec_pod; make test
\ No newline at end of file
+	cd exec_pod; make test
+
+memcheck:
+	cd create_pod; make memcheck; sleep 10
```

Comment:
Why are there sleeps here (and  below)?

Reply:
I want to give the Kubernetes components (e.g. kubelet, API server) some time to pull the image and initialize the pod when creating a pod. Prevent possible errors in the following examples (e.g. deleting the pod, executing command in the pod when the pod is not initialized).

Reply:
Perhaps move the `sleep 10` to it's own line and add a comment?

```
memcheck:
      sleep 10 # wait for kubernetes components to come up
     cd create_pod; make memcheck
      ...

```

wdyt?

Reply:
It's a good idea. And I find a better "waiting" solution:

```bash
kubectl wait --for=condition=ready --all pod -n namespace --timeout=30s
```

Let me have a try today.

Reply:
Done. Please take another look.